app/templates/components/toezicht/inzendingen-table.hbs

```diff
@@ -21,7 +21,13 @@
       {{/unless}}
       <td class="u-visible--tablet-portrait-up" style="max-width: 250px;">{{row.bestuurseenheid.naam}} <span class="u-hidden--tablet-landscape-up text-fade"><br>{{row.bestuurseenheid.classificatie.label}}</span></td>
       <td class="u-visible--tablet-landscape-up">{{row.bestuurseenheid.classificatie.label}}</td>
-      <td style="max-width: 250px;">{{row.besluitType.label}} <span class="u-hidden--tablet-portrait-up text-fade"><br>{{row.bestuurseenheid.naam}} {{row.bestuurseenheid.classificatie.label}}</span></td>
+      <td style="max-width: 250px;">
+        {{row.besluitType.label}} <br>
+        <span class="u-hidden--tablet-portrait-up text-fade">{{row.bestuurseenheid.naam}} {{row.bestuurseenheid.classificatie.label}}</span><br>
+        {{#if (eq row.regulationType.label 'Reglementen en verorderingen')}}
```

Comment:
Condition must be set on `row.decisionType`
It's not a good practice to match on the label since this is volatile. URI is the most stable property of a resource to compare on.

Reply:
I believe I have fixed this issue as well as others. Please take a look.